### PR TITLE
Pulse 4718 - HOTFIX for dogwhistle

### PR DESCRIPTION
**Pull request by andrewkcarter**
This adds the missing counter metrics for dogwhistle.

It also removes an extraneous stdout logger and toggles the 30 second sleep out of the default startup.

## Testing
Add a `traptor.env` file (copy traptor.env.sample) during testing, defining values for:
```
CONSUMER_KEY=
CONSUMER_SECRET=
ACCESS_TOKEN=
ACCESS_TOKEN_SECRET=
```
Then execute:
```
git clone git@github.com:istresearch/traptor.git
cd traptor
git checkout PULSE-4718
virtualenv ~/env
. ~/env/bin/activate
pip install -r requirements.txt
docker-compose up -d --build
```
Once running, go to Graphite at http://localhost:8002 and ensure `stats.traptor.src.heartbeat.success.count` is being recorded.

### CODE REVIEW

File: traptor/traptor.py [hunk 1]
```diff
@@ -1393,7 +1388,7 @@ def createArgumentParser():
     parser = argparse.ArgumentParser()
     parser.add_argument(
             '--skipdelay',
-            action='store_true',  # which defaults to False.
+            action='store_false',  # which defaults to True.
```
Comment (martinezah) [inline traptor/traptor.py]: If I'm reading this properly, we now have a CLI argument `--skipdelay` which *adds* a delay to the startup; if that's correct I think it probably needs to be renamed, eg to `--delay` or something along those lines?

**Comment by martinezah** [comment]
Tests pass; need to add a `traptor.env` file during testing with values for:
```
CONSUMER_KEY=
CONSUMER_SECRET=
ACCESS_TOKEN=
ACCESS_TOKEN_SECRET=
```

File: traptor/traptor.py (hunk 1 above)
Comment (andrewkcarter) [reply traptor/traptor.py]: Yeah, but I wanted to minimize my footprint.

Comment (andrewkcarter) [reply traptor/traptor.py]: I think it needs to go away entirely. I will rename the flag.

**Pull request merged**